pages/form/Question5.js

```diff
@@ -1,29 +1,39 @@
 import { useState } from "react";
-import { Heading, Text, Select } from "@chakra-ui/react";
-import Layout from "../../components/Layout/Layout";
-import useForm from "../../components/FormProvider";
-import { departments } from "../../utils/constants";
-import LinkButton from "../../components/LinkButton/LinkButton";
-import capitalize from "../../utils/capitalize";
-import Q5Progress from "../../public/images/progress-bar/q5-progress-bar.svg";
-import { sendFormResponse } from "../../utils/dbApi";
+import { Heading, Text, Select, Stack, Grid } from '@chakra-ui/react';
+import Layout from '../../components/Layout/Layout';
+import useForm from '../../components/FormProvider';
+import { departments } from '../../utils/constants';
+import LinkButton from '../../components/LinkButton/LinkButton';
+import {
+  BackButton,
+  ContinueButton,
+  SubmitButton,
+} from '../../components/LinkButton/LinkButton';
+import capitalize from '../../utils/capitalize';
+import Q5Progress from '../../public/images/progress-bar/q5-progress-bar.svg';
+import { sendFormResponse } from '../../utils/dbApi';
 
 export default function Question5() {
   const { answers, setAnswers } = useForm();
   const [department, setDepartment] = useState(answers.department);
+  const [incentive, setIncentive] = useState(answers.incentive);
```

Comment:
I don't think we capture anything with `incentive` variable in Question 5; the response for Question 5 should be captured in `department` variable, right?
So I think you can remove this line.
```suggestion
```